src/appendices/hed.md

```diff
@@ -130,23 +179,42 @@ This allows for a proper validation of the HED annotations (for example using th
 }
 ```
 
-If you omit the `HEDVersion` field from the dataset description file, a warning will be generated and any present HED information will be validated using the latest version of the HED schema.
-This is bound to result in problems, and hence, it is strongly RECOMMENDED that the `HEDVersion` field be included when using HED in a BIDS dataset.
+If you omit the `HEDVersion` field from the dataset description file,
```

Comment:
I checked with @happy5214.   If your dataset has HED in it and you don't have a HEDVersion field in your dataset_description.json, an error is generated rather than making an assumption about version.

Reply:
bids-validator@1.12.0 doesn't seem to be validating HED and the [hed-examples Jupyter notebook](https://github.com/hed-standard/hed-examples/blob/main/hedcode/jupyter_notebooks/bids/validate_bids_dataset_with_libraries.ipynb) seems to be validating against the latest version of the HED schema in case the `HEDVersion` field is omitted.
What version should I check this against?